src/evidently/features/is_valid_sql_feature.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any
+from typing import ClassVar
+from typing import Optional
+
+import sqlvalidator
```

Comment:
Let's move this import to the [beginning](https://github.com/evidentlyai/evidently/pull/1332/files#diff-a1bc3e164c7afaa840d5966343c65f5676709a33346a1f84166f9a0eed89f649R32) of the `is_valid_sql` function

Reply:
Changes made. What was causing the issue?

Reply:
Now all the UI checks are being passed in CI 😊

It's a bit tricky: other fix is to add `llm` extras to install step in these tests but `llm` extra at the same time is kinda overkill and unnecessary